gradle-plugin/it/src/it/extension-configuration/expected/expected-config.yml

```diff
@@ -65,20 +65,16 @@ images:
 - name: "registry/image:tag"
   build:
     contextDir: "@endsWith('extension-configuration/src/main/context-dir')@"
-    dockerFile: "@endsWith('extension-configuration/src/main/context-dir/Dockerfile')@"
```

Comment:
Previously, the images that were declared in the context of the KubernetesExtension class were mutated by the DefaultGeneratorManager.

In the current implementation, (unresolved) images passed in to the DefaultGeneratorManager are no longer mutated and are cloned and enriched incrementally instead.

This is reflected here, where the expected config should only show the values passed in by the user in the build.gradle configuration.